packages/firebase_performance/firebase_performance_web/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```

Comment:
please remove updates to this file.

Reply:
This is needed, otherwise the automatic analyze check will fail with the following error:

``` 
firebase_performance_web:
Publishing firebase_performance_web 0.0.1 to https://pub.dartlang.org:
|-- .gitignore
|-- .metadata
|-- CHANGELOG.md
|-- LICENSE
|-- README.md
|-- lib
|   |-- firebase_performance_web.dart
|   '-- src
|       |-- http_metric.dart
|       '-- trace.dart
|-- pubspec.yaml
'-- test
    '-- firebase_performance_web_test.dart
Package validation found the following potential issue:
* ./CHANGELOG.md doesn't mention current version (0.0.1).
  Consider updating it with notes on this version prior to publication.
```

Reply:
nvm I removed the version number from pubspec.yaml and the error went away.